modules/verify-central-cluster-upgrade.adoc

```diff
@@ -11,14 +11,23 @@ After you have upgraded both Central and Scanner, verify that the Central cluste
 .Procedure
 
 * Check the Central logs:
+
++ 
+If you are using {ocp}:
 +
 [source,terminal]
 ----
-$ oc logs -n stackrox deploy/central -c central <1>
+$ oc logs -n stackrox deploy/central -c central 
 ----
-<1> If you use Kubernetes, enter `kubectl` instead of `oc`.
 +
-If the upgrade is successful, you will see output similar to the following:
+If you are using Kubernetes:
++
+[source,terminal]
+----
+$ kubectl logs -n stackrox deploy/central -c central 
+----
+
+* If the upgrade is successful, you will see output similar to the following:
```

Comment:
Suggest replacing this bullet with a `.label` For example:
.Sample output of a successful upgrade